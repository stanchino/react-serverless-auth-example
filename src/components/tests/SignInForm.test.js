import React from "react";
import { SignInForm } from "..";
import { testConnectedContainer } from "./shared-examples";

describe("SignInForm", () => testConnectedContainer(<SignInForm />));