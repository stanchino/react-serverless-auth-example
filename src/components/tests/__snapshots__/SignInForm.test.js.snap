// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm renders without errors 1`] = `
<form
  className={undefined}
  onSubmit={[Function]}
>
  <div
    className="messages"
  >
    <div
      className="flash"
    />
  </div>
  <div>
    <label />
    <div>
      <input
        autoComplete="email"
        name="email"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Email"
        type="text"
        value=""
      />
    </div>
  </div>
  <div>
    <label />
    <div>
      <input
        autoComplete="new-password"
        name="password"
        onBlur={[Function]}
        onChange={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
    </div>
  </div>
  <a
    aria-current={false}
    className={undefined}
    form={
      Object {
        "anyTouched": false,
        "array": Object {
          "insert": [Function],
          "move": [Function],
          "pop": [Function],
          "push": [Function],
          "remove": [Function],
          "removeAll": [Function],
          "shift": [Function],
          "splice": [Function],
          "swap": [Function],
          "unshift": [Function],
        },
        "asyncValidate": [Function],
        "asyncValidating": false,
        "blur": [Function],
        "change": [Function],
        "clearSubmit": [Function],
        "destroy": [Function],
        "dirty": false,
        "dispatch": [Function],
        "error": undefined,
        "form": "signIn",
        "handleSubmit": [Function],
        "initialValues": undefined,
        "initialize": [Function],
        "initialized": false,
        "invalid": false,
        "pristine": true,
        "reset": [Function],
        "submitFailed": false,
        "submitSucceeded": false,
        "submitting": false,
        "touch": [Function],
        "untouch": [Function],
        "valid": true,
        "warning": undefined,
      }
    }
    href="/auth/reset"
    onClick={[Function]}
    style={undefined}
  >
    Forgotten Password?
  </a>
  <button
    disabled={true}
    type="submit"
  >
    Sign In
  </button>
  <button
    disabled={true}
    onClick={[Function]}
    type="reset"
  >
    Cancel
  </button>
</form>
`;
